### PR TITLE
Fix internal slots

### PR DESCRIPTION
**Pull request by relu91**
I noticed that we are not correctly referencing the internal slots we defined. Moreover, in `ExposedThing` we are using internal slots in the "old way". I'd like to fix this issue too.

***
<a href="https://pr-preview.s3.amazonaws.com/relu91/wot-scripting-api/pull/424.html" title="Last updated on Dec 19, 2022, 6:28 PM UTC (b76dcee)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-scripting-api/424/6fcfdb6...relu91:b76dcee.html" title="Last updated on Dec 19, 2022, 6:28 PM UTC (b76dcee)">Diff</a>

### CODE REVIEW

**Review by zolkis — APPROVED** [review]

**Comment by danielpeintner** [comment]
@relu91 mentioned that we need additional work w.r.t. [internal slots for ExposedThing](https://pr-preview.s3.amazonaws.com/relu91/wot-scripting-api/pull/424.html#internal-slots-for-exposedthing) similar to what we have for [ConsumedThing internal slots](https://pr-preview.s3.amazonaws.com/relu91/wot-scripting-api/pull/424.html#internal-slots-for-consumedthing).

**Comment by relu91** [comment]
Todos: 
1. Internal slots for events
2. Check the "subscribers" internal slots cited in the spec and see if it make sense to covert those into the new notation

**Review by zolkis — APPROVED** [review]

**Comment by relu91** [comment]
Ready for review, we agreed to merge it once @danielpeintner approves it. **Reminder**: open two issues with the todos above.

File: index.html [hunk 1]
```diff
@@ -2802,13 +2826,13 @@ <h3>The <dfn>PropertyReadHandler</dfn> callback</h3>
...
           <li>
-            Let |property| be the value of [[\td]]'s |properties|'s |name|.
+            Let |property| be the value of {{ExposedThing/[[td]]}}'s |properties|'s |name|.
             If it does not exist, send back a {{NotFoundError}} in the
             reply and abort these steps.
           </li>
           <li>
-            If there is an [[\unobserveHandler]] of type {{Function}} defined
-            for |name| on |property|, invoke that with |options|, send back a
+            If there is a user provided {{Function}} in {{ExposedThing/[[observeHandlers]]}} <a>internal slot</a>
```
Comment (danielpeintner) [inline index.html:2834]: I think this should be {{ExposedThing/[[unobserveHandler]]}}

**Review by danielpeintner — CHANGES_REQUESTED** [review]
LGTM, I think there is one bug, see comment below

File: index.html (hunk 1 above)
Comment (relu91) [reply index.html]: Good catch, fixed!!

**Review by danielpeintner — APPROVED** [review]
LGTM, Ready to go now...

**Comment by relu91** [comment]
Ok got two approves from the editors, as discussed I'm merging.

**Pull request merged**